### PR TITLE
Code moved from other repo.

### PR DESCRIPTION
**Pull request by JacobKingDev**

### CODE REVIEW

File: DigiMeSDK/Core/Classes/Network/DMERequestFactory.m [hunk 1]
```diff
@@ -91,7 +91,6 @@ - (NSString *)baseUrlPath
 {
     return [NSString stringWithFormat:@"%@%@/permission-access", self.baseUrl, kDigiMeAPIVersion];
 }
-
```
Comment (michaeleustace) [inline DigiMeSDK/Core/Classes/Network/DMERequestFactory.m]: Please revert this change

File: DigiMeSDK/Core/Classes/Utility/NSString+DMECrypto.m [hunk 2]
```diff
@@ -95,7 +95,7 @@ - (NSData *)base64Data
     
     const unsigned char *charString = (const unsigned char *)[self UTF8String];
     
-    NSMutableData *theData = [NSMutableData dataWithCapacity:self.length];
+    NSMutableData *theData = [NSMutableData dataWithCapacity: [self length]];
```
Comment (michaeleustace) [inline DigiMeSDK/Core/Classes/Utility/NSString+DMECrypto.m]: Please revert this change

File: DigiMeSDK/Postbox/Classes/DMEClient+Postbox.m [hunk 3]
```diff
@@ -0,0 +1,84 @@
+//
+//  DMEClient+Postbox.m
+//  DigiMeSDK
+//
+//  Created by Jacob King on 16/10/2018.
```
Comment (michaeleustace) [inline DigiMeSDK/Postbox/Classes/DMEClient+Postbox.m]: Naughty naughty - please remove your name and add copyright statement to this and header file

File: README.md [hunk 4]
```diff
@@ -24,12 +24,12 @@ Digi.me SDK depends on digi.me app being installed to enabled user initiate auth
      * [Handling app callback](#handling-app-callback)
      * [Delegate Calls (authorize)](#delegate-calls-authorize)
   * [Fetching data](#fetching-data)
-     * [Fetching Accounts data](#fetching-accounts-data)
```
Comment (michaeleustace) [inline README.md]: Please revert this change

File: DigiMeSDK/Core/Classes/Security/CADataDecryptor.m [hunk 5]
```diff
@@ -50,19 +50,6 @@ + (NSData *)decrypt:(NSData *)jsonData error:(NSError * _Nullable __autoreleasin
         
         return decryptedData;
     }
-    else
```
[the remark this answers is not in the document]
Comment (JacobKingDev) [reply DigiMeSDK/Core/Classes/Security/CADataDecryptor.m]: This looks as if it was just returning nil in the case that we have unencrypted data; dropping the else should achieve the same?

File: DigiMeSDK/Core/Classes/DMEAppCommunicator.m [hunk 6]
```diff
@@ -0,0 +1,222 @@
...
+- (NSString *)appId
+{
+    return [DMEClient sharedClient].appId;
+}
+
+#pragma mark - SKStoreProductViewControllerDelegate
+
+- (void)productViewControllerDidFinish:(SKStoreProductViewController *)viewController
+{
+
```
[the remark this answers is not in the document]
Comment (JacobKingDev) [reply DigiMeSDK/Core/Classes/DMEAppCommunicator.m]: This now happens in DMEAppCommunicator.m

[the remark this answers is not in the document]
Comment (michaeleustace) [reply DigiMeSDK/Core/Classes/DMEAppCommunicator.m]: This hasn't been resolved - please put a body in the function.

File: DigiMeSDK/Core/Classes/Security/CADataDecryptor.m [hunk 7]
```diff
@@ -50,6 +50,19 @@ + (NSData *)decrypt:(NSData *)jsonData error:(NSError * _Nullable __autoreleasin
         
         return decryptedData;
     }
+    else
+        {
```
Comment (michaeleustace) [inline DigiMeSDK/Core/Classes/Security/CADataDecryptor.m]: Incorrect indentation!
